lib/events/pgevents/common/utils.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgcommon
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/gravitational/trace"
+	"github.com/jackc/pgerrcode"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgconn"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/sirupsen/logrus"
+
+	"github.com/gravitational/teleport/api/utils/retryutils"
+)
+
+// TODO(espadolini): delete in favor of lib/backend/pgbk/common
+
+// ConnectPostgres will open a single connection to the "postgres" database in
+// the database cluster specified in poolConfig.
+func ConnectPostgres(ctx context.Context, poolConfig *pgxpool.Config) (*pgx.Conn, error) {
+	connConfig := poolConfig.ConnConfig.Copy()
+	connConfig.Database = "postgres"
```

Comment:
nit: Is there a constant for that?

Reply:
I can't find a good name for it, even the postgres docs don't give it a name other than "a database named `postgres`, which is meant as a default database for use by utilities, users and third party applications".